mojave/structures/doors.dm

```diff
@@ -18,3 +18,33 @@
 			flick("roomc0", src)
 		if("closing")
 			flick("roomc1", src)
+
+/obj/machinery/door/unpowered/ms13/rustic
+	icon = 'mojave/icons/structure/talldoor.dmi'
+	icon_state = "rustic_closed"
+	name = "log door"
+	opacity = 1
+	density = TRUE
+	explosion_block = 1
+	layer = ABOVE_MOB_LAYER
+
+/obj/machinery/door/unpowered/ms13/rustic/update_icon()
+	if(density)
+		icon_state = "rustic_closed"
+	else
+		icon_state = "rustic_open"
+	if(dir == NORTH)
+		pixel_y = 8
+	if(dir == EAST)
+		pixel_x = -6
+	if(dir == SOUTH)
+		pixel_y = -8
+	if(dir == WEST)
+		pixel_x = 6
```

Comment:
Make this a switch, and also I think this should be getting done in initialize(), I don't think the code needs to be rerun every single time the door is opened or closed, just when the door is initialized.